DataStructuresMemories/ViewControllers/DataStructuresTableViewController.swift

```diff
@@ -10,9 +10,9 @@ import UIKit
 
 
 
-class DataStructuresMemoriesTableViewController: UITableViewController {
+class DataStructuresTableViewController: UITableViewController {
    
-    let dataStructModel = DataStructModel()
+    let dataStructModel = DataStructuresModel()
```

Comment:
It is data source, so could you name it properly?

Reply:
Yes. I will change it right now.